README.md

```diff
@@ -1,65 +1,50 @@
-# React and Redux
+## Bookstore
+The Bookstore is a website similar to the "Awesome Books" website built in the previous module. You will create an MVP version of it that allows you to:
 
-If you are not familiar with linters and GitHub Actions, read [root level README](../README.md).
+Display a list of books.
+Add a book.
+Remove a selected book.
+By building this application, you will learn how to use React and Redux.
 
-## Set-up GitHub Actions
 
-This GitHub Action is going to run [ESLint](https://eslint.org/) and [Stylelint](https://stylelint.io/) to help you find style issues.
+## Built With
 
-[Stylelint](https://stylelint.io/) is a linter for your stylesheets that helps you avoid errors and enforce conventions.
+- HTML
+- CSS
+- JavaScript
+- React
 
-[ESLint](https://eslint.org/) is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs.
+## Live Demo
 
-Please do the following **steps in this order**:
+coming soon
+ 
+## Getting Started
 
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
-    - **Remember** to use the file linked above
-    - **Remember** that `.github` folder starts with a dot.
-2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-    - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub Actions:
+To get a local copy up and running follow these simple example steps.
 
-![gh actions checks](../assets/images/gh-actions-eslint-stylelint-checks.png)
+1. Git clone ggit@github.com:oshanedesign/bookstore.git
 
-Click on the `Details` link to see the full output and the errors that need to be fixed:
+2. Cd Bookstore
 
-![gh actions failing checks](../assets/images/gh-actions-html-css-failing-checks.png)
+3. Run npm start to get project running locally 
```

Comment:
- [ ] I would recommend including the important step here i.e. `npm install` to install all the [packages and dependency. Please include it before `npm start`.

Reply:
Thank you for that suggestion @Dipeshtwis added
![e56](https://user-images.githubusercontent.com/40554384/166067580-aefa1240-7779-46e3-bb2c-d3064f0b178f.PNG)